client/lib/abtest/active-tests.js

```diff
@@ -18,8 +18,9 @@ module.exports = {
 	coldStartReader: {
 		datestamp: '20160804',
```

Comment:
This is a reminder to adjust the `datestamp` to the expected A/B test start date.